sample-code/src/server.ts

```diff
@@ -723,3 +724,28 @@ app.get('/prompt-registry/template', async (req, res) => {
     sendError(res, error);
   }
 });
+
+app.get('/test/deployment-id', async (req, res) => {
+  try {
+    // Example: test with a model name (foundation-models)
+    const deploymentId1 = await getDeploymentId(
+      'gpt-4o',
+      'azure-openai',
+      'foundation-models'
+    );
+
+    // Example: test with a deploymentId config (orchestration)
+    const deploymentId2 = await getDeploymentId(
+      { deploymentId: 'd123456789', resourceGroup: 'my-group' },
```

Comment:
[q] I dont understand this, we call `getDeploymentId` but also pass a deploymentId?